### DISCUSSION TITLE
How do I integrate ezdxf into my .NET app?

### DISCUSSION THREAD

**Discussion by tonyatglazierscenterdotcom** [mozman/ezdxf#1088] [General]
How do I install EZDXF to work with C# and .Net. Do I need to install python to use this ezdxf? I have no experience in Python. Or is there a .dll I can reference?

Tony

**Comment by mozman** [comment]
I have no experience with C# but googling "embed python in c#" returns this:
https://pythonnet.github.io/pythonnet/dotnet.html

**Comment by tonyatglazierscenterdotcom** [comment]
Thank you.

**Comment by mostl33** [comment]
If you don't need rendering (or develop your own "renderer"):

https://github.com/haplokuon/netDxf

It is implemented in C# and is pretty complete (available via NuGet).

Cheers,
ML

**Comment by tonyatglazierscenterdotcom** [comment]
Thanks, yeah that's what I've been looking into.
